modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.tsx

```diff
@@ -58,7 +58,9 @@ const Card = ({item, schema}: {item: any; schema: ICardSchema}) => {
 		?.value;
 	const localizedTitle = getLocalizedValue(item, schema.title)?.value || '';
 	const selectedItemKey = selectedItemsKey && item[selectedItemsKey];
-	const formattedActions = filterItemActions(actionsRef.current, item) as any;
+	const formattedActions =
```

Comment:
Quick fix. Still displaying the three dots icon when DSM do not provide actions but the API endpoint does.